aurora/aurora-war/src/main/java/org/silverpeas/looks/aurora/servlets/GoToSpaceHomepageBackOffice.java

```diff
@@ -0,0 +1,86 @@
+package org.silverpeas.looks.aurora.servlets;
+
+import org.silverpeas.core.admin.component.model.ComponentInst;
+import org.silverpeas.core.admin.component.model.ComponentInstLight;
+import org.silverpeas.core.admin.component.model.Parameter;
+import org.silverpeas.core.admin.component.model.WAComponent;
+import org.silverpeas.core.admin.service.Administration;
+import org.silverpeas.core.util.URLUtil;
+import org.silverpeas.core.util.logging.SilverLogger;
+import org.silverpeas.core.web.http.HttpRequest;
+import org.silverpeas.core.web.look.LookHelper;
+import org.silverpeas.looks.aurora.AuroraSpaceHomePage;
+import org.silverpeas.looks.aurora.LookAuroraHelper;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+public class GoToSpaceHomepageBackOffice extends HttpServlet {
+
+	private static final long serialVersionUID = 1L;
+
+	public void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+		doPost(req, res);
+	}
+
+	public void doPost(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {
+    HttpRequest request = HttpRequest.decorate(req);
+    String spaceId = request.getParameter("SpaceId");
+    LookAuroraHelper lookHelper = (LookAuroraHelper) LookHelper.getLookHelper(req.getSession(false));
+
+    String destination = "admin/jsp/accessForbidden.jsp";
+    if (lookHelper.isSpaceAdmin(spaceId)) {
+      ComponentInstLight backOfficeApp = lookHelper.getConfigurationApp(spaceId);
+      if (backOfficeApp == null) {
+        createConfigurationApp(spaceId, lookHelper.getUserId());
+        backOfficeApp = lookHelper.getConfigurationApp(spaceId);
+      }
+
+      if (backOfficeApp != null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS2583)